typings/common-types.d.ts

```diff
@@ -0,0 +1,167 @@
+declare module 'common-types' {
```

Comment:
Why is this a module instead of a simple `.ts` file that you can import interfaces from?

Reply:
No real reason. I assumed that it made sense for types to be in a .ts' file. Should I update it?

Note that you can't have relative paths in declaration files, so if I wanted to reference something in `unstable.d.ts` I had to have `common-types` the same format.